keyboards/polykybd/tools/compare_codegen.py

```
@@ -0,0 +1,239 @@
+
+    if differing:
+        ok = False
+        print(f"differing bodies: {len(differing)}")
+        import difflib
+
+        for name in differing[: args.show]:
+            b_bodies, c_bodies = base[name], cand[name]
+            if len(b_bodies) != len(c_bodies):
+                print(f"\n--- {name}: emitted {len(b_bodies)} time(s) -> {len(c_bodies)}")
+                continue
+            for idx, (b, c) in enumerate(zip(b_bodies, c_bodies)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)